src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseExpressionBodyForLambda
+{
+    using Microsoft.CodeAnalysis.Text;
+    using static UseExpressionBodyForLambdaHelpers;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal class UseExpressionBodyForLambdaDiagnosticAnalyzer : AbstractCodeStyleDiagnosticAnalyzer
```

Comment:
>UseExpressionBodyForLambdaDiagnosticAnalyzer [](start = 19, length = 44)

sealed (probably the fixer and refactoring too)

Reply:
I don't mind doing this.  However, in general, this is not something the IDE is particular anal about :)